games-misc/opengfx/opengfx-0.6.1.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} )
+
+inherit python-any-r1
+
+DESCRIPTION="OpenGFX data files for OpenTTD"
+HOMEPAGE="https://wiki.openttd.org/en/Basesets/OpenGFX https://github.com/OpenTTD/OpenGFX"
+SRC_URI="https://cdn.openttd.org/${PN}-releases/${PV}/${P}-source.tar.xz"
+S="${WORKDIR}/${P}-source"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+
+DEPEND="
+	games-util/grfcodec
+	games-util/nml
+	${PYTHON_DEPS}
+"
+
+DOCS=( "README.md" "changelog.txt" )
+
+src_prepare() {
+	default
+
+	python-any-r1_pkg_setup
+}
+
+src_compile() {
+	myemakeargs=(
+		GIMP=""
+		PYHTON="${EPYTHON}"
```

Comment:
typo?

Reply:
Huh, that funny. It comes from already existing 0.6.0. Fixed now.